src/components/BoardSection.jsx

```diff
@@ -15,9 +19,17 @@ const BoardSection = ({title, data, $addWork, plus}) => {
           ))
         ) : null}
         {$addWork ? (
-        <div className='card__addwork' onClick={plus ? plus : null}>
-          <GoPlusCircle size={28} />
-        </div>
+        <>
+        {isOpen ? (
```

Comment:
[Suggestion]
&&을 사용해서 $addWork가 false일때 undefined를 반환하게 하면 더 깔끔할거 같아요.
```suggestion
        {isOpen && (
            <NewBoard id={id} closeOpen={closeOpen} />
          )}
```

Reply:
[commit-b9da59e]